packages/jellycommands/src/JellyCommands/commands/types/commands/Command.ts

```diff
@@ -3,16 +3,23 @@ import type { ApplicationCommandOptionData } from 'discord.js';
 import { ApplicationCommandType } from 'discord-api-types/v9';
 import type { CommandInteraction } from 'discord.js';
 import { BaseCommand } from '../../base/BaseCommand';
-import { schema, CommandOptions } from './options';
+import { schema, CommandOptions, AutocompleteHandler } from './options';
 import { ApplicationCommand } from 'discord.js';
 import { removeKeys } from 'ghoststools';
 
 export class Command extends BaseCommand<CommandOptions, CommandInteraction> {
+    autocomplete?: AutocompleteHandler;
```

Comment:
Might be good to add `private readonly` access modifier on here

Reply:
How is it supposed to be called with a private 👀

Reply:
Ah `public readonly`